README.rst

```diff
@@ -27,25 +27,28 @@ First, you connect to Scrapinghub::
 You can list the projects available to your account::
 
     >>> conn.project_ids()
-    [u'123', u'456']
+    [123, 456]
 
 And select a particular project to work with::
 
-    >>> project = conn['123']
+    >>> project = conn[123]
```

Comment:
I would have left an example using string since the most common task is passing the project id from the arguments of a script.